frontends/ol-search-ui/src/components/ExpandedLearningResourceDisplay.test.tsx

```diff
@@ -1,75 +1,90 @@
 import React from "react"
-import { render, screen, fireEvent, waitFor } from "@testing-library/react"
-import { assertInstanceOf } from "ol-util"
-import ExpandedLearningResourceDisplay from "./ExpandedLearningResourceDisplay"
+import { render, screen, fireEvent, within } from "@testing-library/react"
+import user from "@testing-library/user-event"
+import { faker } from "@faker-js/faker"
+import { assertInstanceOf, assertNotNil } from "ol-util"
+import { getByTerm, queryByTerm } from "ol-util/build/test-utils"
+import { makeUrl } from "ol-util/build/factories"
+import LearningResourceDetails, {
+  LearningResourceDetailsProps
+} from "./ExpandedLearningResourceDisplay"
 import {
-  makeMinimalResoure,
+  makeCourseResult,
   makeImgConfig,
+  makeRun,
   makeVideoResult
 } from "../factories"
-import { resourceThumbnailSrc, getInstructorName } from "../util"
+import { resourceThumbnailSrc, getInstructorName, findBestRun } from "../util"
+
+const formatShareLink: LearningResourceDetailsProps["formatShareLink"] = r =>
+  `www.tests.org?resource_id=${r.id}&resource_type=${r.object_type}`
+
+const renderLearningResourceDetails = (
+  overrides: Partial<LearningResourceDetailsProps> = {}
+): LearningResourceDetailsProps => {
+  const resource = makeCourseResult()
+  const imgConfig = makeImgConfig()
+  const props = { resource, imgConfig, formatShareLink, ...overrides }
+  render(<LearningResourceDetails {...props} />)
+  return props
+}
 
 describe("ExpandedLearningResourceDisplay", () => {
-  it("renders the expected fields", () => {
-    const resource = makeMinimalResoure()
-    const imgConfig = makeImgConfig({
-      embedlyKey: SETTINGS.embedlyKey,
-      ocwBaseUrl: SETTINGS.ocw_next_base_url,
-      width:      440,
-      height:     239
-    })
-    render(
-      <ExpandedLearningResourceDisplay
-        object={resource}
-        runId={resource.runs[0].id}
-        setShowResourceDrawer={null}
-      />
-    )
-
-    expect(screen.getByText(resource.title)).toBeInTheDocument()
+  it("renders the expected fields for a course", () => {
+    const run = makeRun()
+    const offerer = "ocw"
+    const resource = makeCourseResult({ runs: [run], offered_by: [offerer] })
+
+    const { imgConfig } = renderLearningResourceDetails({ resource })
 
     const instructors = resource.runs[0].instructors
       .map(instructor => getInstructorName(instructor))
       .join(", ")
-    expect(screen.getByText(instructors)).toBeInTheDocument()
```

Comment:
I changed most of the `expect(screen.getBy...).toBeInTheDocument()` to just `screen.getBy...`.

From the [`testing-library`](https://testing-library.com/docs/queries/about/#types-of-queries) docs:
> Returns the matching node for a query, and throw a descriptive error if no elements match or if more than one match is found